TramsDataApi/Gateways/EstablishmentGateway.cs

```diff
@@ -17,6 +18,14 @@ public Establishment GetByUkprn(string ukprn)
         { 
             return _dbContext.Establishment.FirstOrDefault(e => e.Ukprn == ukprn);
         }
+        public IEnumerable<int> GetURNsByRegion(IEnumerable<string> regions)
```

Comment:
It looks like you're following convention here but I'm surprised to see these aren't async.

Reply:
I agree it is strange. Happy to make the change if you think it's worth it?